p4-16/spec/P4-16-spec.mdk

```diff
@@ -3489,6 +3489,10 @@ Bit-strings also support the following operations:
   The effect of this statement is to set bits `H` through `L` (inclusive of
   both) of `e` to the bit-pattern represented by `x`, and leaves all other bits
   of `e` unchanged.  A slice of an unsigned integer is an unsigned integer.
+  A slice may also be specified as `[L+:W]` where L is the lowest bit of the slice
+  and W is the width of the slice.  In this case, only W must be a non-negative compile-time
```

Comment:
Should we restrict `W` to a local compile-time known value, like the spec currently does for both `H` and `L` for the current slices, if I recall correctly?

Reply:
yes